lib/sequenceserver/database.rb

```diff
@@ -168,12 +168,11 @@ def make_blast_database(file, type)
         _make_blast_database(file, type, title, taxid)
       end
 
-      def _make_blast_database(file, type, title, taxid, quiet = false)
```

Comment:
Using `sys` changes the behaviour of this method to `quiet = true` by default. Here, we want the output of `cmd` to be printed to the console.

The fourth argument is provided to silence the output of this method during testing.